spec/factories/users.rb

```diff
@@ -7,6 +7,7 @@
     email { Faker::Internet.email }
     password { Faker::Internet.password }
     confirmed_at { DateTime.now }
+    last_sign_in_at { 1.day.ago.change(usec: 0) }
```

Comment:
Ez elméletileg egy nullable field. Én nem raknám bele a factory-ba, ha csak nem ad valamilyen plusz hasonlóságot az éles adatokhoz hogy benne van. Lefutnak a tesztek e nélkül is?

Reply:
Ha benne szeretnéd hagyni a factory-ban, akkor érdemes lenne a többi fake-elt adatot átnézni, hogy koherens-e velük. A fenti kód:
```ruby
    confirmed_at { DateTime.now }
    last_sign_in_at { 1.day.ago.change(usec: 0) }
```
Valószínűleg nem fordul elő valós futás + adatok mellett.

Reply:
Csináljunk belőle trait-et.